test/CMakeLists.txt

```diff
@@ -85,19 +85,24 @@ foreach(_test Callbacks Degenerate ManufacturedSolution ComparisonWithBoost)
     "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp" COPYONLY
   )
   list(APPEND ARBORX_TEST_QUERY_TREE_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BVH.cpp")
-  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp.tmp"
-    "#include <ArborX_BruteForce.hpp>\n"
-    "#define ARBORX_TEST_TREE_TYPES Tuple<ArborX::BruteForce>\n"
-    "#define ARBORX_TEST_DEVICE_TYPES std::tuple<${ARBORX_DEVICE_TYPES}>\n"
-    "#define ARBORX_TEST_DISABLE_NEAREST_QUERY\n"
-    "#define ARBORX_TEST_DISABLE_CALLBACK_EARLY_EXIT\n"
-    "#include <tstQueryTree${_test}.cpp>\n"
-  )
-  configure_file(
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp.tmp"
-    "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp" COPYONLY
-  )
-  list(APPEND ARBORX_TEST_QUERY_TREE_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/tstQueryTree${_test}_BF.cpp")
+  foreach(_bounding_volume Box)
```

Comment:
Why did you change this?

Reply:
True, I did not have to do it in this PR. This is in preparation for the future testing of BruteForce with multi-dimensional geometries.

Reply:
Yes

Reply:
@dalg24 Done.